packages/ProgressBar/src/ProgressBar.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import Heading from "@paprika/heading";
+import * as sc from "./ProgressBar.styles";
+
+const propTypes = {
+  /** Descriptive a11y text for assistive technologies. */
+  a11yText: PropTypes.string,
+  /** Description for the ProgressBar */
+  bodyText: PropTypes.string,
+  /** Header text for the ProgressBar */
+  header: PropTypes.string,
+  /** Specifies how much progress has been completed from 0-100 */
+  completed: PropTypes.number,
+};
+
+const defaultProps = {
+  a11yText: null,
+  bodyText: null,
+  header: null,
+  completed: 0,
+};
+
+function ProgressBar(props) {
+  const { a11yText, header, bodyText, completed, ...moreProps } = props;
+
+  const bestAria = a11yText || "Loading";
+
+  return (
+    <sc.ProgressBar data-pka-anchor="progress-bar" {...moreProps}>
```

Comment:
I would expect the `data-pka-anchor` to be on the actual progress bar and not the container. What do you think?

Reply:
Yes, good point. I'll update this one 👍